quality-assurance/Security-Testing-Guidance.md

```diff
@@ -0,0 +1,116 @@
+# Overview
+This document is intended to provide some guidance as to what level of testing should be considered within the project when 
+tackling security testing, and is aimed at the Shift Left approach to test early. This will look at the OWASP as a source of 
+the test area that needs to be covered.
+
+# Types of testing
+Following the should be considered within the project / application with regards to security testing
+<ul>
+	<li>Access Control Management
+		<p>We need to baseline this off the least access as a default. Testing the systems access control will cover 
+		<ul>
+			<li>Logic for user aproval/li>
+			<li>Ensure that the trust boundaries are clearly defined</li>
+			<li>Deny by default</li>
+			<li>Ensure that session access is enforced for every request</li>
+			<li>Ensure that Authentication is enforced on statis resources</li>
+		</ul></p>
+	</li>	
+	<li>Static analysys 
+	<p>part of the white box testing this is best performed as suggested in OWASP as part of the code review
+		This is best suited for the multitude of tooling currently on the market
+	</p></li>
+	<li>Server Access Control
+	<p>In order to test this then the tester should,
```

Comment:
Are there example web pages you can link out to for the items below so that someone can read exactly how to do each one?